Dockerfile

```diff
@@ -83,6 +83,9 @@ RUN curl -L https://istio.io/downloadIstio | ISTIO_VERSION="$ISTIO_VERSION" sh -
 ARG K3D_VERSION=4.4.8
 RUN curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v$K3D_VERSION bash
 
+RUN curl -fsSL https://raw.githubusercontent.com/ahmetb/kubectl-aliases/master/.kubectl_aliases -o /etc/profile.d/kubectl_aliases.sh && \
```

Comment:
If there is a better source of aliases that folks know about, I am happy to use whichever